adsbibdesk.py

```diff
@@ -1330,9 +1330,12 @@ def filetype(filename):
                         is not None:
                     fd, pdf = tempfile.mkstemp(suffix='.pdf')
                     cmd = 'ssh -p %s %s@%s \"touch /tmp/adsbibdesk.pdf; ' \
-                        'wget -O /tmp/adsbibdesk.pdf \\"%s\\"\"' % \
+                        'wget -O /tmp/adsbibdesk.pdf ' \
+                        '--header=\\"Accept: text/html\\" ' \
+                        '--user-agent=\\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36\\" ' \
+                        '\\"%s\\"\"' % \
```

Comment:
For MNRAS articles, the proxy method failed, because the redirected link seen from a machine outside the paywall doesn't contain any PDF information. Here we will just let wget on the proxy follow the ads links directly.